packages/@ourworldindata/grapher/src/barCharts/DiscreteBarChart.tsx

```diff
@@ -457,7 +457,7 @@ export class DiscreteBarChart
                                 y={0}
                                 transform={`translate(${labelX}, 0)`}
                                 fill="#555"
-                                dominantBaseline="central"
+                                dominantBaseline="middle"
```

Comment:
For me, it actually seems that `central` works better (in Chrome, at least).
Is there a particular reason why you changed it back to `middle`?

Reply:
I think I got worried about browser support for no particular reason. You're right though – I'll add it back in and if we run into issues it's easy to revert :)